package.json

```diff
@@ -40,6 +40,7 @@
         "cookie-parser": "1.4.5",
         "cors": "2.8.5",
         "cronosjs": "1.7.1",
+        "crypto-js": "^4.0.0",
```

Comment:
You have not added this dependency to any of the package.json files under `packages/node_modules/...`, so it won't be included in the published versions of the modules.

Reply:
Added it to `runtime/package.json`.